### ISSUE TITLE
ColorKeyword: Color `grey` should be written in hexadecimal form as `#808080`

### ISSUE THREAD

**Issue by douglasdeodato** [brigade/scss-lint#830]
I have this code here: 

```
// ----
// Sass (v3.4.21)
// Compass (v1.0.3)
// ----

$grey: #808080;

// Call the color palette modifiers in color values
@function palette($palette, $shade: 'base') {
  @return map-get(map-get($color-palettes, $palette), $shade);
}


$color-palettes: (
		grey: (
		xx-light : lighten($grey, 43%),
		x-light  : rgba(240,240,240, 1),
	  light    : rgba(221, 221, 221, 1),
	  base     : $grey,
	 	dark     : rgba(133, 133, 133, 0.24),
	 	x-dark   : darken($grey, 16%)
	  ),
);



.black-palette {
    background:palette(grey);
   
}
```

`[W] ColorKeyword: Color `grey` should be written in hexadecimal form as `#808080`
`

this is not a error the variable was defined with hexadecimal colour? is scss lint supporting sass palettes?

if I am right this suppose not be a error.

I know I can disable this on my scss-lint.yml, but I would like you if will be a option to avoid in this case.

thanks all

**Comment by sds** [comment]
Hey @douglasdeodato,

Please ensure you are running the latest version of SCSS-Lint (0.49.0 at time of writing). This was fixed in 0.39.0. 

0.38.0 was the last version published as `scss-lint`, so make sure you `gem install scss_lint` (with an underscore) instead. The name change was so SCSS-Lint switched to proper [Rubygems naming conventions](http://guides.rubygems.org/name-your-gem/).

Hope that helps!

**Comment by douglasdeodato** [comment]
thank you.
